lib/rbac/filterer.rb

```diff
@@ -351,9 +351,11 @@ def get_self_service_objects(user, miq_group, klass)
 
     def calc_filtered_ids(scope, user_filters, user, miq_group, scope_tenant_filter)
       klass = scope.respond_to?(:klass) ? scope.klass : scope
+      expression = miq_group.try(:entitlement).try(:filter_expression)
```

Comment:
You probably don't need the `try` for `entitlement`. In practice, every group should have an entitlement. Since removing the `try` will cause tests to fail, this can be done in a followup PR.